.travis.yml

```diff
@@ -0,0 +1,18 @@
+sudo: required
+
+language: minimal
+
+services: docker
+
+branches:
+    only:
+        - master
+
+script:
+    # conventional image name, but no intention to push it so far
+    - docker build -t liberoadmin/patterns-core .
+
+after_success:
+    - docker run --name patterns-core liberoadmin/patterns-core node_modules/.bin/gulp exportPatterns
```

Comment:
Gulp is run twice in this case? (Once in `build`, and again here.)

Reply:
Could be, if there is a dependency expressed in the task

Reply:
Changing to not re-run `gulp build`